### ISSUE TITLE
Write a Cassovary shell

### ISSUE THREAD

**Issue by lintool** [twitter/cassovary#212]
Just like there's a Spark shell for interactively playing around, we should write a Cassovary shell that support interactive manipulation of the graphs. I've started prototyping something in my clone, under `repl` branch.

Better yet, instead of the current server, we should have a backend server process that loads a potentially large graph, and multiple frontend shells can connect to it and access the graph. We could use the same mechanism that Spark uses for serializing closures for execution on the backend.

Thoughts?

**Comment by pankajgupta** [comment]
Yep that will be awesome. Memory map work done by Peter makes it easier to
load large graphs too...
